code/modules/supermatter/supermatter.dm

```diff
@@ -24,6 +24,34 @@
 
 #define WARNING_DELAY 20			//seconds between warnings.
 
+//This is really gross and I hate doing it like this.
```

Comment:
I am still bewildered by why you think an orderly list of sounds is gross.
It's fine.
This comment is gross though.